docs/src/main/tut/typeclasses/functor.md

```diff
@@ -6,137 +6,106 @@ source: "core/src/main/scala/cats/Functor.scala"
-`F[B]`.  The name of the method `map` should remind you of the `map`
-method that exists on many classes in the Scala standard library, for
-example:
+A `Functor` instance must obey two laws:
 
-```tut:book
-Option(1).map(_ + 1)
-List(1,2,3).map(_ + 1)
-Vector(1,2,3).map(_.toString)
-```
+* Composition: Mapping with `f` and then again with `f` is the same as mapping once with the composition of `f` and `g`
+    * `fa.map(f).map(g) = fa.map(f.andThen(g))`
 
-## Creating Functor instances
+* Identity: Mapping with the identity function is a no-op
+    * `fa.map(x => x) = fa`
 
-We can trivially create a `Functor` instance for a type which has a well
-behaved `map` method:
+## A different view
 
-```tut:silent
-import cats._
+Another way of viewing a `Functor[F]` is that `F` allows the lifting of a pure function `A => B` into the effectful
+function `F[A] => F[B]`. We can see this if we re-order the `map` signature above.
 
-implicit val optionFunctor: Functor[Option] = new Functor[Option] {
-  def map[A,B](fa: Option[A])(f: A => B) = fa map f
-}
+```tut:book:silent
+trait Functor[F[_]] {
+  def map[A, B](fa: F[A])(f: A => B): F[B]
 
-implicit val listFunctor: Functor[List] = new Functor[List] {
-  def map[A,B](fa: List[A])(f: A => B) = fa map f
+  def lift[A, B](f: A => B): F[A] => F[B] =
+    fa => map(fa)(f)
 }
 ```
 
-However, functors can also be created for types which don't have a `map`
-method. For example, if we create a `Functor` for `Function1[In, ?]`
-we can use `andThen` to implement `map`:
+## Functors for effect management
 
-```tut:silent
-implicit def function1Functor[In]: Functor[Function1[In, ?]] =
-  new Functor[Function1[In, ?]] {
-    def map[A,B](fa: In => A)(f: A => B): Function1[In,B] = fa andThen f
-  }
-```
+The `F` in `Functor` is often referred to as an "effect" or "computational context." Different effects will
+abstract away different behaviors with respect to fundamental functions like `map`. For instance, `Option`'s effect
+abstracts away potentially missing values, where `map` applies the function only in the `Some` case but
+otherwise threads the `None` through.
 
-This example demonstrates the use of the
-[kind-projector compiler plugin](https://github.com/non/kind-projector).
-This compiler plugin can help us when we need to change the number of type
-holes. In the example above, we took a type which normally has two type holes, 
-`Function1[?,?]` and constrained one of the holes to be the `In` type, 
-leaving just one hole for the return type, resulting in `Function1[In,?]`. 
-Without kind-projector, we'd have to write this as something like 
-`({type F[A] = Function1[In,A]})#F`, which is much harder to read and understand.
+Taking this view, we can view `Functor` as the ability to work with a **single** effect - we can apply a pure
+function to a single effectful value without needing to "leave" the effect.
 
-## Using Functor
+## Functors compose
```

Comment:
now this section looks like this

## Functors Compose
we can compose either with compose or with nested
compose example
nested example
compose notes
nested notes

how about this instead?
## Functors Compose
we can compose either with compose or with nested
### compose 
compose example
compose notes
### nested
nested example
nested notes